### ISSUE TITLE
Error message for multigene graph

### ISSUE THREAD

**Issue by beamilon** [IGS/gEAR#211]
The genes are now available but I get an error message when trying to create a graph. I tried Heatmap, Dotplot and Violin and get the the message. Of note, I got that message for P2, mouse, scATAC-seq, Organ of Corti Transcription Factor Motif Accessibility (Waldhaus) but it was only for the heatmap and I was able to create a dotplot.
4dpf, zebrafish, RNA-seq, TU-tagged hair cells mRNA and whole larvae input (Nicolson)
![image](https://user-images.githubusercontent.com/70719873/153242249-63a91d16-c597-46e2-b347-9a4ff68bb5c3.png)

**Comment by adkinsrs** [comment]
For one of them, I actually printed this error to the error logs (and it should have been propagated to the curator page itself)

```raise PlotError("Only one gene from the searched gene symbols was found in dataset.  The heatmap option require at least 2 genes to plot.")```

Did you attempt to create a heatmap with just a single gene?

**Comment by beamilon** [comment]
No. I used 3 genes, Gfi1aa, Gfi1ab, Sox2

**Comment by beamilon** [comment]
The genes don't appear in the drop down list if they are not found in the dataset and these three were good.

**Comment by adkinsrs** [comment]
Ok, I'll poke around then.  It may actually be a different error then.  That error I referenced was actually meant for displaying plots after a front page search (where genes may or may not be in the dataset).

**Comment by adkinsrs** [comment]
I see the error now when using the zebrafish dataset

` AttributeError: 'float' object has no attribute 'lower'`

Easy fix to make.  Interestingly enough, I tried the Waldhaus dataset on my local copy of gEAR and got another error related to returning the gene symbols... @beamilon did this happen to you?

**Comment by beamilon** [comment]
I really don't remember what genes I used exactly. It's normal not to find the first two because they are specific to zebrafish. I may have used: Sox2, Pou4f3, pou3f4, Gfi1 (but not 100% sure)

**Comment by adkinsrs** [comment]
<img width="1373" alt="Screen Shot 2022-02-10 at 10 14 43 AM" src="https://user-images.githubusercontent.com/5665914/153437278-b5bb767b-e00d-4812-91d0-559ed8f13d83.png">

Pou4f3 was not in that dataset, so I did select some genes at random and created this. So I wonder if it is a specific combination of genes that leads to the error.  Given that I have found some genes from the zebrafish dataset that lead to some issues, that could be the case.
